pgzero/runner.py

```diff
@@ -102,6 +103,8 @@ def prepare_mod(mod):
     set before the module globals are run.
 
     """
+    fn_hash = hash(mod.__file__) % ((sys.maxsize + 1) * 2)
```

Comment:
I think this hash might need to be stronger than this. Also I don't believe `hash()` is necessarily stable across Python interpreter versions. I'd suggest using `hashlib.sha1()`.